eve/tests/single-node/init.sh

```diff
@@ -13,7 +13,3 @@ if systemctl is-enabled --quiet iptables; then
 fi
 iptables -F || die
 iptables -X || die
-
```

Comment:
With kubernetes worker this is not strictly needed anymore. But I find it better like that, so I let that part